client/src/components/container/Billing/Billing.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { PremiumForm, } from '../../presentation/presentation.js';
+import { Elements, StripeProvider, } from 'react-stripe-elements';
+import './style/css/Billing.css';
+
+class Billing extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+
+    };
+
+  }
+
+  render() {
+    return (
+      <div className='billing-container'>
+        <StripeProvider className='billing-container' apiKey='pk_test_5S7b3sU9cayM2p0lyZCGZR5e00mmeHbCML'>
```

Comment:
For the time being, having an api key will work here but I would recommend putting it in another file that does not get uploaded onto github as you get closer towards a finishing product.